README.md

```diff
@@ -6,87 +6,66 @@ Deis (pronounced DAY-iss) is an open source PaaS that makes it easy to deploy an
 applications on your own servers. Deis builds on [Kubernetes](http://kubernetes.io/) to provide
 a lightweight, [Heroku-inspired](http://heroku.com) workflow.
```

Comment:
sure, fixed in https://github.com/deis/builder/commit/7bd8437ad7d6f0c5d7b8f6fcde55089586cc3220. Let me know what you think about the text I replaced it with.